### PR TITLE
Added android support to cordova-gimbal-sdk

### PR DESCRIPTION
**Pull request by koga73**
Added android sdk and updated plugin config for install

### CODE REVIEW

**Comment by rlepinski** [comment]
Awesome.

Everything looks generally ok from a first glance. Instead of using the jar files, could we instead use [gradle/maven](https://docs.gimbal.com/android/v2/devguide.html#add_android_studio_maven_central.)? 

You then can supply a`.gradle` file instead of jars. Example - https://github.com/urbanairship/phonegap-ua-push/blob/master/src/android/build-extras.gradle

It will make updating Gimbal's SDK easier in the future.

File: plugin.xml [hunk 1]
```diff
@@ -30,4 +30,45 @@
         <framework src="CoreLocation.framework" />
         <framework src="libz.dylib" />
     </platform>
-</plugin>
+    
+    <!-- android -->
+    <platform name="android">
+		<!-- Gimbal background processes -->
+		<config-file target="AndroidManifest.xml" parent="/manifest/application">
```
Comment (rlepinski) [inline plugin.xml:37]: We can remove this once we move to gradle based install

**Comment by koga73** [comment]
I'm not very familiar with gradle, however based on your example it seems it's used during compilation and I'm not sure how it would apply in this case. The Gimbal SDK can only be downloaded from within the Gimbal user portal and comes as 3 jar files. I don't believe they have the source code hosted anywhere.

**Comment by rlepinski** [comment]
Gimbal distributes an .aar file through maven. This file includes the gimbal jar files and the manifest entries. 

Instead of the manifest changes and jar files, you can add a build-extras.gradle file with:

```
repositories {
    mavenCentral()
}

dependencies {
    compile group: 'com.gimbal.android.v2', name: 'gimbal-sdk', version: '+'
    compile group: 'com.gimbal.android.v2', name: 'gimbal-slf4j-impl', version: '+'
}
```

Then include it in the plugin.xml file instead of the jar files.

**Comment by rlepinski** [comment]
Nvm, it looks like they only distribute the gimbal jars, not an aar file. So you can still do the gradle file above, but keep the manifest changes.

**Comment by koga73** [comment]
Done! Removed the jar files in the repo in place of using gradle.

File: README.md [hunk 2]
```diff
@@ -1 +1,7 @@
 # cordova-gimbal
+
+## Installation
+
+```xml
+	cordova plugin add urbanairship-gimbal-sdk-cordova
```
Comment (rlepinski) [inline README.md]: We were unable to add the plugin to NPM due to an issue with how the iOS framework is distributed. That is why we moved gimbal-sdk into its own repo.  Should be `cordova plugin add https://github.com/urbanairship/cordova-gimbal-sdk`

**Comment by rlepinski** [comment]
Minor README change then Ill merge it. Great job!

**Comment by koga73** [comment]
Ah. Updated readme!

**Comment by rlepinski** [comment]
Thanks!

**Pull request merged**